.github/workflows/loadgen.yaml

```diff
@@ -0,0 +1,31 @@
+name: Generate Load on crAPI
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+jobs:
+  loadgen:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: levoai/actions/test-plan@v1-beta
+        with:
+          # Authorization key required to execute the Levo CLI. Please refer to https://app.levo.ai/settings/keys to get your authorization key.
+          authorization-key: '${{ secrets.LEVO_CLI_AUTH_KEY }}'
+
+          # The ID of your organization in Levo dashboard. Please refer to https://app.levo.ai/settings/organization to get your organization id.
+          organization-id: '5f733ed8-39c9-40d1-86b3-11e2b0451842'
+
+          # URL for the instance of the application under test.
+          target: 'http://crapi.levo.ai'
```

Comment:
@ricekot this cluster is different from the cluster where we're installing the latest sensor and Satellite. You can probably get the IP of that service and use that here.